Sources/MapboxMaps/Foundation/BaseMapView.swift

```diff
@@ -94,9 +94,11 @@ open class BaseMapView: UIView, MapClient, MBMMetalViewProvider, CameraViewDeleg
     private var observerConcrete: ObserverConcrete!
     @objc dynamic internal var displayLink: CADisplayLink?
 
-    @IBInspectable var styleURI__: String = ""
-    @IBInspectable var baseURL__: String = ""
-    @IBInspectable var accessToken__: String = ""
+    @IBInspectable internal var styleURI__: String = ""
```

Comment:
Preexisting, so no need to make any changes for this PR, but why does this variable have two trailing underscores?

Reply:
These may no longer be required, but IIRC it's so that the type can be String (IB won't show the trailing underscores).